.gitlab-ci.yml

```diff
@@ -215,6 +215,20 @@ run-tests:
     - job: image-build-tests-runner
     - job: image-build-runner
 
+boardfarm-tests:
+  stage: test
+  image: $CI_REGISTRY_IMAGE/prplmesh-boardfarm-ci:$CI_PIPELINE_ID
+  script:
+    - export USER=gitlab-runner BFT_DEBUG=y
+    - tests/run_bf.sh
+  artifacts:
+    paths:
+      - logs
+  needs:
+    - job: build-in-docker
+  tags:
+    - boardfarm
```

Comment:
The commit message says something completely different...

> It runs in a shell runner

No, it doesn't have the shell tag and it specifies an image.

> It also requires docker to be available on the runner, so add the
"docker-build" tag.

No, it doesn't have the docker-build tag.

Reply:
I changed it back and forth when I noticed the shell runner had the issue with the user permissions.
It's a docker executor now, I edited the commit message.